content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-widgets-init.js

```diff
@@ -59,6 +59,29 @@
             $field.prop("checked", $field.attr("value") === value);
         },
 
+        isHiddenField: function ($field) {
+            return !_.isEmpty($field) && ($field.prop("type") === "hidden");
+        },
+
+        setHiddenField: function($field, value) {
+            var $parent = $field.parent();
+            if ($parent.hasClass("coral-DatePicker")) {
```

Comment:
@acheson0410 can you add functions like isDateField(), isRichTextField() and setDateField(), setRichTextField() for making them reusable